README.adoc

```diff
@@ -137,10 +180,39 @@ endif::[]
 
 The [hotspot=localConnector]`localConnector` feature is required by the Arquillian Liberty Managed container to connect to and communicate with the Open Liberty runtime. The [hotspot=Servlet]`servlet` feature is required during the deployment of the Arquillian tests in which servlets are created to perform the in-container testing.
 
+Open another command-line session and run the `configure-arquillian` goal at the `start` directory to integrate Arquillian and the Arquillian Liberty managed and remote containers with your existing project.
+
+[role="command"]
+```
+mvn liberty:configure-arquillian
+```
+
+Because you started Open Liberty in dev mode, all the changes were automatically picked up. You can run the tests by pressing the `enter/return` key from the command-line session where you started dev mode. You will see the following output:
```

Comment:
```suggestion
Because you started Open Liberty in dev mode, all the changes were automatically picked up. You can run the tests by pressing the `enter/return` key from the command-line session where you started dev mode. Look for the following output:
```